src/components/Tooltip/sass/Tooltip.scss

```diff
@@ -1,10 +1,12 @@
 .tyk-tooltip {
+  --tooltip-max-width: 450px;
```

Comment:
Wondering if it would make sense to have this as a common variable here 
(https://github.com/TykTechnologies/tyk-ui/blob/master/src/common/sass/variables.scss) ???

Reply:
I think I will create a variables file for the component and put it in there. Similar like we have for other components.